src/components/ui/StaticFileMetadata.vue

```diff
@@ -67,8 +69,14 @@ export default class StaticFileMetadata extends Vue {
     return !!this.ffaListing ? this.ffaListing.license : ''
   }
 
-  public get fileSize(): number {
-    return !!this.ffaListing ? this.ffaListing.size : 0
+  public get fileSize(): string {
+    if (!this.ffaListing) { return '' }
+    return FileHelper.fileSizeString(this.ffaListing.size)
```

Comment:
Can use ternary: `return !this.ffaListing ? '' : FileHelper.fileSizeString(this.ffaListing.size)`